starwars/src/App.js

```diff
@@ -24,12 +27,17 @@ class App extends Component {
       });
   }
   render() {
+    this.stateItems = this.state.starwarsChars.map((item, i) =>
```

Comment:
I'm not opposed to seeing this pattern. I'm not sure I love it, but I'm not opposed to this. I've seen many other ways that components can map their data. Some would say that the "render()" function should be completely pure, and call functions/methods to handle this logic for them. I say just use the `{}` interpolating method inside your react JSX to map over data and load it to the page. Nice work! I like what i"m seeing so far!